spec/folio_client/authenticator_spec.rb

```diff
@@ -7,49 +7,69 @@
   let(:okapi_headers) { {some_bogus_headers: "here"} }
   let(:token) { "a_long_silly_token" }
   let(:connection) { FolioClient.configure(**args).connection }
+  let(:http_status) { 200 }
+  let(:http_body) { "{\"okapiToken\" : \"#{token}\"}" }
```

Comment:
These are the bits that vary below, so we give them names instead of having lots of mostly repetitive stub requests in before blocks, which obscures the bits that vary.